examples/nemo/scripts/KGOs/namelist_cfg_orca2_like

```diff
@@ -0,0 +1,226 @@
+!!>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
+!! NEMO/OPA  BENCH Configuration namelist : overwrite some defaults values defined in SHARED/namelist_ref
```

Comment:
Sorry this may be a misunderstanding of mine, I thought that for this idealised case ORCA2 just mean changing the problem size of the file that came in that folder. What namelist_cfg should I be using? This is the same that was on the BENCH_PASSTHROUGH in the previous integration test.

And since we are discussing the the test configuration, I was also wondering if (at least for passthrough) it is worth re-enabling the "key_iomput key_top" as these delete during preprocessing some code, which decreases the coverage of the passthrough. Would it make sense to enable them to expose as many bugs as possible? (even if they are not part of NGARCH performance study)

Reply:
Yes, ORCA2_ICE is more than just BENCH with a different problem size (although I couldn't tell you what the differences are exactly). We should use the 'standard' namelist for this configuration which probably lives in the NEMO repo.
However, I agree that for pass-through, the more code we see, the better so I would be happy to enable those two keys. Perhaps as a second PR if they don't 'just work'.

Reply:
It turns out that BENCH already had ICE on by default and the IO and XIOS off by default (the ice is also set to SI3 in the namelist_cfg). So I just deleted the keys to make it more explicitly "default" BENCH. However, I am not using ORCA2 sizes anymore, I am using something smaller in order to run 100 iterations in reasonable time. So I end up calling the files `...BENCH.small...

The ORCA2_ICE_PISCES is a distinct test case from BENCH which we could also add (and indeed has more components), but is not idealised, so it needs model files and I imagine we can not change the size so easily, so I would way until we are further along in optimising NEMOv5:
```
./makenemo
Please choose at least one reference configuration (-r) or testcase (-a) from:

  ¤ Reference configurations with default sub-components (can be updated by a new set)
AGRIF_DEMO           OCE ICE TOP NST
...
ORCA2_SAS_ICE        OCE ICE NST SAS
ORCA2_ICE_PISCES     OCE TOP ICE NST ABL
SPITZ12              OCE ICE
WED025               OCE ICE

  ¤ Demonstrations cases (see https://github.com/sflavoni/NEMO-test-cases for more)
CANAL                OCE
...
VORTEX               OCE NST
WAD                  OCE
BENCH                OCE ICE TOP
```

Reply:
OK. That seems clear. Hopefully we are approaching/have reached the configuration that Bristol are working with. Either way, this is a big step forward.